### PR TITLE
Add geovista

### PR DESCRIPTION
**Pull request by tkoyama010**
This proposal is to add [GeoVista](https://github.com/bjlittle/geovista) to the Geospatial section.

### CODE REVIEW

File: tools/tools.yml [hunk 1]
```diff
@@ -373,6 +373,12 @@
       badges: pypi
       builtons: [matplotlib]
 
+    - repo: bjlittle/geovista
+      site: https://geovista.readthedocs.io
+      conda_channel: conda-forge
+      badges: pypi, conda, codecov, site
+      builtons: [pyvista]
```
Comment (maximlt) [inline tools/tools.yml:380]: Hey @tkoyama010, if you want pyvista to appear nicely in the `Built on` column of the table you could add its entry to the `builtons.yml` file with its logo in the folder `doc/_static/badges/builtons`.

Comment (tkoyama010) [reply tools/tools.yml:380]: Great idea! I will make it.

Comment (tkoyama010) [reply tools/tools.yml:380]: resolved in 8a98d5d and 7f610a9 .

**Comment by maximlt** [comment]
LGTM, thanks @tkoyama010 !

**Pull request merged**